pkg/cmd/server/api/v1/conversions.go

```diff
@@ -308,6 +312,21 @@ func addConversionFuncs(scheme *runtime.Scheme) {
 			return nil
 		},
 		api.Convert_resource_Quantity_To_resource_Quantity,
+		// TODO: https://github.com/kubernetes/kubernetes/issues/24381
+		func(in *bool, out **bool, s conversion.Scope) error {
```

Comment:
Just imagine how much fun it was to debug that.

On Sun, Apr 24, 2016 at 11:49 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/conversions.go
> <https://github.com/openshift/origin/pull/8474#discussion_r60860721>:
>
> > @@ -308,6 +312,21 @@ func addConversionFuncs(scheme *runtime.Scheme) {
> >  			return nil
> >  		},
> >  		api.Convert_resource_Quantity_To_resource_Quantity,
> > +		// TODO: https://github.com/kubernetes/kubernetes/issues/24381
> > +		func(in *bool, out **bool, s conversion.Scope) error {
>
> My head hurts now
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8474/files/57cd55c93511284b52da60dc754284797c003935#r60860721>
>